spec/controllers/widgets_controller_spec.rb

```diff
@@ -5,8 +5,6 @@
 RSpec.describe WidgetsController, type: :controller, versioning: true do
   before { request.env["REMOTE_ADDR"] = "127.0.0.1" }
 
-  after { RequestStore.store[:paper_trail] = nil }
```

Comment:
Current attributes are automatically reset before each request so this is no longer necessary for the new implementation.